src/services/document-service.spec.js

```diff
@@ -324,7 +324,10 @@ describe('document-service', () => {
     let result;
 
     beforeEach(async () => {
-      await Promise.all(testDocs.map(testDoc => createTestDocument(container, user, testDoc)));
+      for (const testDoc of testDocs) {
```

Comment:
I'm not sure that this solves the root cause though...

Reply:
Please read the Jira ticket first, I tried to make it clear there what the line of investigation was and why this fix is so.

Reply:
It's clear, thanks!